traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyItemMappingFactory.java

```diff
@@ -69,8 +69,13 @@ public CatalogItem instantiate(JsonReader reader) throws Exception {
                     reader.endArray();
                     break;
                 case "geoLocation":
-                    coordinate = new Coordinate.CoordinateObjectMappingFactory()
-                            .instantiate(reader);
+                    JsonToken geoLocationToken = reader.peek();
```

Comment:
Should follow the same format that is used for `thumbnail` to keep it consistent.